src/components/PostDetails.tsx

```diff
@@ -126,16 +53,13 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
                   <a href={`mailto:${comment.email}`} data-cy="CommentAuthor">
                     {comment.name}
                   </a>
-
                   <button
                     data-cy="CommentDelete"
                     type="button"
                     className="delete is-small"
                     aria-label="delete"
-                    onClick={() => deleteComment(comment.id)}
-                  >
-                    delete button
-                  </button>
+                    onClick={() => dispatch(removeComment(comment.id))}
```

Comment:
The function passed to the onClick handler will be recreated every time the component re-renders, which could lead to performance issues if there are a lot of comments. Consider using the useCallback hook to memoize the function.